scipy/signal/tests/test_wavelets.py

```diff
@@ -79,6 +79,29 @@ def test_morlet(self):
         y = wavelets.morlet(20000, w=7, s=20, complete=False)[5000:15000]
         assert_array_almost_equal(x, y, decimal=2)
 
+    def test_morlet2(self):
+        w = wavelets.morlet2(1.0, 0.5)
+        expected = (np.pi**(-0.25) * np.sqrt(1/0.5)).astype(complex)
+        assert_array_equal(w, expected)
+
+        lengths = [5, 11, 15, 51, 101]
+        for length in lengths:
+            w = wavelets.morlet2(length, 1.0)
+            assert_(len(w) == length)
+            max_loc = np.argmax(w)
+            assert_(max_loc == (length // 2))
```

Comment:
What happens for even? Is it just not clear which value will be the true max (because there are two duplicates)? If so this can be checked, too, by checking membership of `np.argsort(w)[::-1][:2]`

Reply:
Looks like this has been accounted for in the modified test now